config/locales/gobierto_dashboards/views/en.yml

```diff
@@ -30,24 +29,57 @@ en:
         summary:
           contracts: Assigned contracts
           contracts_for: contracts for a total amount of
-          label_half_spendings_contracts: <strong><span id="half-spendings-contracts-pct"></span></strong>
-            of contracts accumulate the 50% of all the spending
-          label_larger_contract_amount: The largest contract means a <strong><span
-            id="larger-contract-amount-pct"></span></strong> of all the spending
-          label_less_than_1000: <strong><span id="less-than-1000-pct"></span></strong>
-            of contracts are less than <strong>1.000 €</strong>
+          label_half_spendings_contracts_1: ""
```

Comment:
Could be another option declare variables? For instance:

yml: `%{value} of contracts accumulate...`
vue: `<p class="description" v-html="I18n.t(LABEL, { value: <strong>VARIABLE</strong> } )">`

Just asking

Reply:
I tried that and it didn't work. Maybe I did something wrong.